presto-main/src/main/java/com/facebook/presto/sql/analyzer/Scope.java

```diff
@@ -31,7 +31,9 @@
 import static com.facebook.presto.sql.analyzer.SemanticExceptions.missingAttributeException;
```

Comment:
The commented code was only used in `computeOutputScope` for `QuerySpecification`

In master `FROM` source scope was parent of `QuerySpecification` scope. Both `WithQueries` and fields of source scope were resolvable (though fields weren't `local` in planner). It seems that the parent visitor (e.g: `visitTableSubquery`) always hides `QuerySpecification` scope (it exposes only `RelationType`), so `FROM WithQueries` weren't leaking upward.

Anyway, I run some queries on Presto master and it seems that `WithQueries` are only visible in child AST nodes and they don't leak upward. This is modeled in presto by creating `With` scopes. `createOutputScope` was exposing too much, but as I mentioned it works because of parent visitor that hides `WithQueries`